docs/reference/remap/append.cue

```diff
@@ -0,0 +1,56 @@
+package metadata
+
+remap: functions: append: {
+	arguments: [
+		{
+			name:        "value"
+			description: "The array"
+			required:    true
+			type: ["array"]
+		},
+		{
+			name:        "item"
+			description: "The items to append"
+			required:    true
+			type: ["array"]
+		},
+	]
+	return: ["array"]
+	category: "Array"
+	description: """
+		Adds the specified array to the end of an array and returns the resulting array. The items
+		can be of any VRL type and are added even if items with the same value are already present
+		in the array.
```

Comment:
Is this documentation up-to-date with the new implementation of `append` (e.g. append the items of one array to another)?

I find it rather confusing to read right now.

For example:

> Adds the specified array to the end of an array and returns the resulting array

It doesn't add the array, it appends the _elements_ of the array to the other array. If you want to add an array to an existing array, you'd use `push`.